docs/sphinx/source/whatsnew/v0.11.3.rst

```diff
@@ -12,12 +12,24 @@ Enhancements
 ~~~~~~~~~~~~
 
 
+Bug Fixes
+~~~~~~~~~
+* Ensure proper tz and pytz types in pvlib.location.Location. To ensure that
+  the time zone in pvlib.location.Location remains internally consistent
+  if/when the time zone is updated, the tz attribute is now the single source
+  of time-zone truth, is the single time-zone setter interface, and its getter 
+  returns an IANA string. (:issue:`2340`, :pull:`2341`)
```

Comment:
```suggestion
* The pvlib.location.Location.tz attribute can be initialized with an `int` or
  a whole-number `float`. (:issue:`2340`, :pull:`2341`)
```
I really don't think this is the place to describe how the interface is changed.

Reply:
Seems to me like the "bug fix" here is not to do with what types are allowed, but rather that `Location` now ensures consistency between its `tz` and `pytz` attributes.

Reply:
It's both. My memory is already fading, but I'm pretty sure a non-string value could be assigned to `pvlib.location.Location.tz`, and in addition to that this value and `pvlib.location.Location.pttz` could diverge.

Reply:
My remaining concern here is to capture the reasoning behind a breaking interface change somewhere for posterity. If this can live somewhere else, I'm all for it.